src/identity/components/GlobalHeader/GlobalHeader.tsx

```diff
@@ -78,9 +78,13 @@ export const GlobalHeader: FC = () => {
   }, [orgsList])
 
   const shouldLoadDropdowns = Boolean(activeOrg?.id && activeAccount?.id)
-  const shouldLoadAvatar = Boolean(
-    user?.firstName && user?.lastName && user?.email && org?.id
-  )
+
+  const shouldLoadAvatar = Boolean(user?.email && org?.id)
+  const avatarFirstName =
+    typeof user?.firstName === 'string' ? user.firstName.trim() : ''
```

Comment:
This is to be really explicit that the user's names will be strings. If they're not, coerce an empty string. Added `trim()` because, otherwise, the user avatar won't render both initials if a user added whitespace at the beginning or end of their name fields.